server/configs/application.properties

```diff
@@ -125,6 +125,7 @@ management.server.port=@@shutdownPort@@
 #jsonaccesslog.condition-if=attributeName
 #jsonaccesslog.condition-unless=attributeName
 
+#### Example enforced CSP for local development
```

Comment:
This isn't just an example. It gets deployed.

Reply:
oops. I'd meant to remove that in my stuff.

Reply:
That gets deployed only locally or with the environment variable?

Reply:
It gets deployed locally with the `useLocalBuild` gradle property. It gets deployed to production always.